src/Compilers/CSharp/Test/Symbol/Symbols/Source/ModifierTests.cs

```diff
@@ -169,5 +171,170 @@ struct S<T> where T : struct
+            Diagnostic(ErrorCode.ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn, "readonly").WithLocation(3, 19));
+        }
+
+        [Fact, WorkItem(63758, "https://github.com/dotnet/roslyn/issues/63758")]
+        public void ReadonlyParameter3()
+        {
+            CreateCompilation(@"
+public class Base {
+    public void M(readonly int X) {
+    }
+}").VerifyDiagnostics(
+                // (3,19): error CS9068: 'readonly' is not supported as a parameter modifier.  Did you mean 'in'?
+                //     public void M(readonly int X) {
+                Diagnostic(ErrorCode.ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn, "readonly").WithLocation(3, 19));
+        }
+
+        [Fact, WorkItem(63758, "https://github.com/dotnet/roslyn/issues/63758")]
+        public void ReadonlyParameter4()
+        {
+            CreateCompilation(@"
+public class Base {
+    void M()
+    {
+        var v = (readonly int i) => { };
+    }
+}").VerifyDiagnostics(
+                // (5,18): error CS9068: 'readonly' is not supported as a parameter modifier.  Did you mean 'in'?
+                //         var v = (readonly int i) => { };
+                Diagnostic(ErrorCode.ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn, "readonly").WithLocation(5, 18));
+        }
+
+        [Fact, WorkItem(63758, "https://github.com/dotnet/roslyn/issues/63758")]
+        public void ReadonlyParameter5()
+        {
+            CreateCompilation(@"
+public class Base {
+    void M()
+    {
+        var v = (ref readonly int i) => { };
+    }
+}").VerifyDiagnostics(
+                // (5,22): error CS9068: 'readonly' is not supported as a parameter modifier.  Did you mean 'in'?
+                //         var v = (ref readonly int i) => { };
+                Diagnostic(ErrorCode.ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn, "readonly").WithLocation(5, 22));
+        }
+
+        [Fact, WorkItem(63758, "https://github.com/dotnet/roslyn/issues/63758")]
+        public void ReadonlyParameter6()
+        {
+            CreateCompilation(@"
+public class Base {
+    void M()
+    {
+        var v = (readonly ref int i) => { };
+    }
+}").VerifyDiagnostics(
+                // (5,18): error CS9068: 'readonly' is not supported as a parameter modifier.  Did you mean 'in'?
+                //         var v = (readonly ref int i) => { };
+                Diagnostic(ErrorCode.ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn, "readonly").WithLocation(5, 18));
+        }
+
+        [Fact]
+        public void RefExtensionMethodsNotSupportedBefore7_2_InSyntax()
+        {
+            var code = @"
+public static class Extensions
+{
+    public static void Print(in this int p)
+    {
+        System.Console.WriteLine(p);
+    }
+}
+public static class Program
+{
+    public static void Main()
+    {
+        int p = 5;
+        p.Print();
+    }
+}";
+
+            CreateCompilation(code, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7_1)).VerifyDiagnostics(
```

Comment:
Are these also just moved? If not, `TestOptions.Regular.WithLanguageVersion`

Reply:
twas a move :)